x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/described_form_field.tsx

```diff
@@ -3,24 +3,91 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import { get } from 'lodash';
+import React, { FunctionComponent, useState } from 'react';
+import {
+  EuiDescribedFormGroup,
+  EuiDescribedFormGroupProps,
+  EuiSpacer,
+  EuiSwitch,
+  EuiSwitchProps,
+} from '@elastic/eui';
 
-import React, { FunctionComponent } from 'react';
-import { EuiDescribedFormGroup, EuiDescribedFormGroupProps } from '@elastic/eui';
+import { UseField, ToggleField, useFormData } from '../../../../shared_imports';
 
-import { ToggleableField, Props as ToggleableFieldProps } from './toggleable_field';
+interface SwitchProps extends Omit<EuiSwitchProps, 'checked' | 'onChange'> {
+  initialValue?: boolean;
+  path?: string;
+  keepChildrenMounted?: boolean;
```

Comment:
I believe the prop `keepChildrenMounted` is duplicated and the one in `Props` type is actually used in the code.